web_client/components/containers/upload.jsx

```diff
@@ -5,7 +5,7 @@ import Upload from '../body/upload';
 import actions from '../../actions';
 import { rest } from '../../globals';
 import objectReduce from '../../utils/object-reduce';
-import { Promise } from '../../utils/promise';
+import { Promise, SCHEDULES, map as promiseMap } from '../../utils/promise';
```

Comment:
The "map" name is the most frank and straightforward description for the function while still being unique within the namespace in the promise utility module.  This is the same rule-of-thumb I use when naming most other exported members.  It just so happens that there are plenty of other "map"s (including `Promise.map`, which I'd rather not monkey patch over).

I suspect that in most cases, client modules importing the symbol will choose to give it a more descriptive name within their namespace, but that isn't enough reason to preempt them in this regard.  Indeed, client modules could choose to import all the symbols like so:

    import * as promiseUtils

Clearly in this case, `map` is the ideal name, as it's resulting fully qualified name remains descriptive without being redundant.

This question could just as easily be one of why the symbols are imported the way they are.  My reasoning on this matter boils down to the fact that `Promise` is used everywhere, so I could either type `promiseUtils.Promise` ad nauseam, import the symbols in two steps (`import ... ; let Promise = ...;`), or actually take advantage of an ES6 language feature for its intended purpose.  I figured the latter was the best among admittedly sub-optimal options.